src/index.ts

```diff
@@ -55,7 +55,7 @@ class APODWidget extends Widget {
   /**
    * Handle update requests for the widget.
```

Comment:
Thanks! Can you also update the comment above the function? "Handle activate requests..."